tests/.gitignore

```diff
@@ -1 +1,2 @@
 node_modules
+out
```

Comment:
Tests now create some output here for debugging.

Reply:
nit: can we call this `test-output` or something like that?